SU-SWS/stanford_profile_helper
# D8CORE-2600: visually hide duration dropdown in smartdate fields

# READY FOR REVIEW

# Summary
- We want to visually hide the `Duration` drop-down in Smartdate field widgets.

# Needed By (Date)
- :shrug:

# Urgency
- not critical.

# Steps to Test

1. Check out this branch.
2. Clear your caches.
3.  Go to create an Event.
4. Observe the `Duration` drop down no longer appears in the smartdate field.


# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
